srv/components/ha/cortx-ha/config.sls

```diff
@@ -15,23 +15,17 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-include:
-  - components.ha.cortx-ha.install
 
 Run cortx-ha post_install:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/ha/conf/setup.yaml', 'ha:post_install')
-    - require:
-      - Install cortx-ha
 
 
 {% if "primary" in pillar["cluster"][grains["id"]]["roles"] %}
 
 Run cortx-ha config:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/ha/conf/setup.yaml', 'ha:config')
-    - require:
```

Comment:
Let's maintain these require statements to ensure installation.
Any specific reason this is removed here?

Reply:
please check the above response